bitnami/wordpress/templates/deployment.yaml

```diff
@@ -107,10 +106,68 @@ spec:
               name: wordpress-data
               subPath: wordpress
         {{- end }}
+        - name: prepare-base-dir
+          image: {{ include "wordpress.image" . }}
+          imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
+          {{- if .Values.resources }}
+          resources: {{- toYaml .Values.resources | nindent 12 }}
+          {{- else if ne .Values.resourcesPreset "none" }}
+          resources: {{- include "common.resources.preset" (dict "type" .Values.resourcesPreset) | nindent 12 }}
+          {{- end }}
+          {{- if .Values.containerSecurityContext.enabled }}
+          securityContext: {{- include "common.compatibility.renderSecurityContext" (dict "secContext" .Values.containerSecurityContext "context" $) | nindent 12 }}
+          {{- end }}
+          command:
+            - /bin/bash
+          args:
+            - -ec
+            - |
+              #!/bin/bash
+
+              . /opt/bitnami/scripts/liblog.sh
+
+              info "Copying base dir to empty dir"
+              # In order to not break the application functionality (such as upgrades or plugins) we need
+              # to make the base directory writable, so we need to copy it to an empty dir volume
+              cp -r --preserve=mode /opt/bitnami/wordpress /bitnami
+              info "Copy operation completed"
+          volumeMounts:
+            - name: empty-dir
+              mountPath: /bitnami
+              subPath: app-base-dir
```

Comment:
why `/bitnami`? Maybe I'm biased as Bitnami developer but that directory has been our traditional persistent directory but here we're using it to hold a copy of the original WordPress source code.

Reply:
Changed it